### ISSUE TITLE
64-bit

### ISSUE THREAD

**Issue by svnpenn** [malensek/3RVX#73]
Please add 64-bit version, thank you.

**Comment by joaodforce** [comment]
Why?

**Comment by svnpenn** [comment]
Why not?

**Comment by joaodforce** [comment]
i dont belive the 32-bit architeture is limiting it in any way, but i belive that compiling a 64-bit version would be easy, since the source code is avaible, just download Visual studio community, open the project and compile it to 64bit.

**Comment by svnpenn** [comment]
> just download Visual studio community

This is pretty arrogant. You dont "just" download Visual Studio. Visual Studio
is 3.7 GB:

http://microsoft.com/download/details.aspx?id=48146

I would rather download a 1.8 MB zip of 3RVX 64-bit that is already compiled.
Im not sure many people are going to have the time or patience to do what you
are proposing.

**Comment by joaodforce** [comment]
My point is, not many people need a 64-bit of it, you didnt make a good argument on why do you need it, if you did then it would make sense to build it, but since you are the only one asking for it, you might as well do it yourself.

**Comment by svnpenn** [comment]
Youre right, we dont need 64-bit. We should just downgrade everything back to 32-bit.

**Comment by joaodforce** [comment]
Man I'm so done arguing with you, you look like a child wishing for a lollipop. 
 
This software barely uses 3 Mb of RAM. 
 
The 64-bit architecture would allow it to address more ram, and perform 64-bit operations, that honestly it would not do anyway. 
 
Running a 32-bit application on a 64-bit OS doesn't reduce its performance in any way, so there is no point in building one, it would just make it that the developer would need to support 2 versions of the software making it more problematic to develop, while getting no benefic whatsoever. 
 
Case closed.

**Comment by svnpenn** [comment]
You are right, these projects should all switch back to 32-bit only:

- Windows
- Linux
- 7-Zip
- Firefox
- MPC-HC
- Notepad2

**Comment by joaodforce** [comment]
Dude, you are not bringing nothing of value to the table, look at what you comparing 3RVX to, Windows and Linux are operating systems that clearly benefic from more than 4GBs of ram, and also need to Address al the ram that the system has in order to provide it to other software, Firefox is a browser, you can easily surpass 4 gbs with a few tabs, MPC-HC is a video player, that uses encoding so it clearly benefits from 64-bit instructions, and Notepad2 imagine having dozens of big source files to edit, those applications really benefit from 64-bit.

If you make a good argument on how 3RVX would benefit form beeing 64-bit, i would built you one mysef.

**Comment by svnpenn** [comment]
> you are not bringing nothing

**Comment by joaodforce** [comment]
I'm trying to be reasonable, and you are just trying to be childish and stupid, this conversation is over.

**Comment by svnpenn** [comment]
> you are just trying to be childish and stupid

**Comment by svnpenn** [comment]
https://github.com/frostwire/frostwire/issues/199

**Comment by svnpenn** [comment]
https://github.com/hadouken/hadouken/issues/228

**Comment by svnpenn** [comment]
https://github.com/irzyxa/Volume2/issues/3

**Comment by svnpenn** [comment]
https://github.com/visualfc/liteide/issues/883
